src/drf_yasg/inspectors/field.py

```diff
@@ -105,14 +129,25 @@ def make_schema_definition(serializer=field):
                     if child.required and not getattr(child_schema, 'read_only', False):
                         required.append(property_name)
 
-                result = SwaggerType(
-                    # the title is derived from the field name and is better to
-                    # be omitted from models
-                    use_field_title=False,
-                    type=openapi.TYPE_OBJECT,
-                    properties=properties,
-                    required=required or None,
-                )
+                # the title is derived from the field name and is better to
+                # be omitted from models
+                if description:
+                    result = SwaggerType(
+                        use_field_title=False, 
+                        type=openapi.TYPE_OBJECT, 
+                        properties=properties, 
+                        required=required or None, 
+                        description=description
+                    )
+                
+                # Don't set a description if it's an empty string or None
+                else:
+                    result = SwaggerType(
+                        use_field_title=False,
+                        type=openapi.TYPE_OBJECT,
+                        properties=properties,
+                        required=required or None
+                    )
```

Comment:
@PaulWay SwaggerType is, in fact, `drf_yasg.openapi.Schema`. Maybe it's better to insert description separately?

```python
if description:
    result.description = description
```

Another way:
```python
result = SwaggerType(
    ...
    description=description or None
)
```